### ISSUE TITLE
Make Dropdowns (LanguageDropdown etc) keyboard and SR accessible

### ISSUE THREAD

**Issue by t3chguy** [vector-im/riot-web#11279]
Should be a case of marking it a dropdown and its children as options, currently, the activator is a button and the options are divs with images which inherit the semantics of the activator button.

- [x] handle scrolling when selected element is not in view
- [x] add appropriate aria labels to options
- [x] add aria-selected={true} to active option
- [x] manage focus properly using activedescendent
- [ ] arrow keys should loop around
